utils/generateReadme.ts

```diff
@@ -4,14 +4,7 @@ const sfcTypeSupportDoc = [
   '',
   '## Type Support for `.vue` Imports in TS',
   '',
-  'TypeScript cannot handle type information for `.vue` imports by default, so we replace the `tsc` CLI with `vue-tsc` for type checking. In editors, we need [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin) to make the TypeScript language service aware of `.vue` types.',
-  '',
-  "If the standalone TypeScript plugin doesn't feel fast enough to you, Volar has also implemented a [Take Over Mode](https://github.com/johnsoncodehk/volar/discussions/471#discussioncomment-1361669) that is more performant. You can enable it by the following steps:",
-  '',
-  '1. Disable the built-in TypeScript Extension',
-  "    1) Run `Extensions: Show Built-in Extensions` from VSCode's command palette",
-  '    2) Find `TypeScript and JavaScript Language Features`, right click and select `Disable (Workspace)`',
-  '2. Reload the VSCode window by running `Developer: Reload Window` from the command palette.',
+  'TypeScript cannot handle type information for `.vue` imports by default, so we replace the `tsc` CLI with `vue-tsc` for type checking.',
```

Comment:
I think we should keep a reference to Volar here: 
```
  'TypeScript cannot handle type information for `.vue` imports by default, so we replace the `tsc` CLI with `vue-tsc` for type checking. In editors, we need [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar) to make the TypeScript language service aware of `.vue` types.',
```

Reply:
Nice catch! I've added back Volar reference.